code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -4,6 +4,8 @@
 // crit percent
 #define MALF_AI_ROLL_CRIT_CHANCE 5
 
+var/turrets_upgraded = FALSE //If the turrets are upgraded
```

Comment:
Unmanaged global, this also probably shouldn't be a global

Reply:
Implemented change to a global. Reasoning for the global is that I have a datum/AI_Module/ and a datum/action/ both referencing or mutating this variable.